### PR TITLE
[WIP] add MT into key feature

### PR DESCRIPTION
**Pull request by butsugiri**
This PR adds description of MT into the `Key Features`.

I wonder if I should put MT results in the README, too (given the results that we have right now).

### CODE REVIEW

**Comment by sw005320** [comment]
Can you add some additional MT results section in https://github.com/espnet/espnet/blob/7be9be92259df37af5e21534f1623ce5057f30a9/README.md#results-and-demo ?
You can just make a table and put the pre-trained models similar to the ASR result.

I want to authorize your result.

**Comment by butsugiri** [comment]
OK, I added results

File: README.md [hunk 1]
```diff
@@ -546,11 +548,26 @@ Available pretrained vocoder models in the demo script are listed as follows:
...
+|    Model    | test2012 | test2013 | test2014 | Pretrained Model |
+|:-----------:|:--------:|:--------:|:--------:|:----------------:|
+| Transformer |   32.07  |   33.52  |   29.13  |       [link](https://drive.google.com/open?id=1o150rdEhPubN1i36TK_dWktFkxuL-X19)       |
+
 
 ## Chainer and Pytorch backends
 
 |                    | Chainer                         | Pytorch                            |
 | -----------        | :----:                          | :----:                             |
+
```
Comment (sw005320) [inline README.md]: This line seems to break the table...

**Comment by sw005320** [comment]
Can you also fix the conflict? @hirofumi0810 is also preparing the document.

**Comment by butsugiri** [comment]
I tried to fix the conflict and now it seems that I made a huge mistake.
Let me close this PR and create another PR.

**Pull request closed without merging**